apps/extension/src/ui/apps/dashboard/routes/Networks/NetworksPage.tsx

```diff
@@ -101,19 +42,32 @@ export const NetworksPage = () => {
       withBack
       backTo="/settings/networks-tokens"
     >
-      <HeaderBlock
-        title={t("Ethereum Networks")}
-        text={t("Add or delete custom Ethereum networks")}
-      />
+      <HeaderBlock title={t("Manage Networks")} text={t("View, edit and delete custom networks")} />
       <Spacer large />
       <div className="flex justify-end gap-4">
+        <ProviderTypeSwitch
```

Comment:
I think ethereum should be the default tab as it's tweaked often, whereas most people have no reason to go tweak substrate chains